lib/contentful-util.js

```diff
@@ -2,7 +2,7 @@ const { documentToHtmlString } = require('@contentful/rich-text-html-renderer');
 const pkg = require('../package.json');
 
 function normalizeEntries({ contentTypes, entries, defaultLocaleCode, options }) {
-    return entries.map(entry => normalizeEntry({ contentTypes, entry, defaultLocaleCode, options })).filter(entry => entry);
+    return entries.map(entry => normalizeEntry({ contentTypes, entry, defaultLocaleCode, options })).filter(Boolean);
```

Comment:
Weird why it's helping as nulls are still filtered out with `entry => entry`
![image](https://user-images.githubusercontent.com/2149612/168736685-20051880-cce9-4aba-bbb3-a3eaf5810d32.png)